### ISSUE TITLE
Ignore elements with `translate=no` attribute

### ISSUE THREAD

**Issue by jelmervdl** [browsermt/bergamot-translator#401]
See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/translate

I can make elements with this attribute skip into the skip-element function, which is used for ignoring `<code>` etc. But that might mess up sentences, as this attribute can also apply to named entities. Those would, when the skip-element trick is used, completely disappear from the sentence when it is submitted to the translator, which might not like missing words (or names).

**Comment by kpu** [comment]
Yeah in general the MT system would need to be trained with a placeholder to do this correctly.
